engine/runtime/src/main/java/org/enso/interpreter/runtime/callable/function/Function.java

```diff
@@ -360,6 +360,16 @@ public static MaterializedFrame getLocalScope(Object[] arguments) {
     }
   }
 
+  @ExportMessage
+  Type getMetaObject(@CachedLibrary("this") InteropLibrary thisLib) {
+    return EnsoContext.get(thisLib).getBuiltins().function();
```

Comment:
why not just `getType`?

Reply:
Because `getType` accepts `TypesLibrary` and I have just `InteropLibrary` at hand. I would need to create a 3rd method and let both message methods delegate to it.